### PR TITLE
feat: update postlight parser to 2.2.3

### PR DESCRIPTION
**Pull request by haynesgt**
Updates to latest version, as 2.2.0 was years old.

The original project was renamed from postlight/mercury to just postlight/parser

Besides this I just tested 50 rss / extension sites and it seems like readability.js is great for about 85% of rss/extension pages whereas postlight/mercury is great for 70-75% of such pages, with postlight generally never succeeding when readability fails. (tested with the chrome extensions). [data here](https://docs.google.com/spreadsheets/d/1TvVhzbdTw8G4B1l0uz-sfDKYEVayNPUsNQAw1NO03Xk/edit#gid=583001697). Maybe we can consider that in the future

### CODE REVIEW

File: routes.js [hunk 1]
```diff
@@ -9,9 +9,12 @@ router.route('/').get((req, res) => {
 });
 
 router.route('/parser').get(async (req, res) => {
-    if (!('x-api-key' in req.headers) || req.headers['x-api-key'] !== process.env.TOKEN) {
-      res.status(401)
-      return res.json({message: '401 - Not Authenticated'})
+    if (
+        !('x-api-key' in req.headers) ||
+        req.headers['x-api-key'] !== process.env.TOKEN
```
Comment (milani) [inline routes.js:14]: now that you are making changes, do you mind using time safe equal? https://nodejs.org/api/crypto.html#cryptotimingsafeequala-b

**Review by milani — CHANGES_REQUESTED** [review]

File: routes.js (hunk 1 above)
Comment (haynesgt) [reply routes.js]: ok. I'll also add npm audit fix

**Review by milani — APPROVED** [review]
Can we test it without going to prod first? In other words, do we have a separate deployment for staging? looks like the URL is hard-coded in metaurl 🤔

**Comment by haynesgt** [comment]
No, we don't deploy in staging I did test it locally. Maybe we can set up a staging deployment.

**Pull request merged**